packages/landing/src/components/layout.header.tsx

```diff
@@ -178,12 +179,26 @@ Your Service/App URL:
 
     const children: ReactNode[] = [];
     if (projections.has(EpsgCode.Nztm2000)) {
-      const nztmTileUrl = Config.map.toTileUrl(MapOptionType.Wmts, Nztm2000QuadTms);
+      const nztmTileUrl = Config.map.toTileUrl(
```

Comment:
NIT: I think this needs to change into some other way of constructing a tileUrl.

Using a object I think would make this a lot more understandable.
`Config.map.toTileUrl({ type: Wmts, tileMatrix: GoogleTms })`